newa/__init__.py

```diff
@@ -369,6 +369,45 @@ def from_yaml_url(cls: type[SerializableT], url: str) -> SerializableT:
         r = get_request(url=url, response_content=ResponseContentType.TEXT)
         return cls.from_yaml(r)
 
+    @classmethod
+    def from_yaml_with_include(cls: type[SerializableT], location: str) -> SerializableT:
+
+        def load_data_from_location(location: str,
+                                    stack: Union[list[str], None] = None) -> dict[str, Any]:
```

Comment:
`Union[list[str], None]` is basically `Optional[list[str]]`

Reply:
I tried `Optional` previously but I was getting some error. Now I am not getting an error so clearly I was doing something wrong. Anyway, fixed.